### PR TITLE
Disable repo metadata signature check on SUSE/SLES

### PR DESCRIPTION
**Pull request by hush-hush**
**Why**

This fix fresh install on SUSE/SLES. It's also align Ansible and the install script from the website.

### CODE REVIEW

File: tasks/pkg-suse.yml [hunk 1]
```diff
@@ -24,31 +24,30 @@
...
-  zypper_repository:
-    name: datadog_5
-    description: Datadog, Inc.
-    repo: "{{ datadog_agent5_zypper_repo }}"
-    enabled: yes
-    auto_import_keys: yes
-    disable_gpg_check: "{% if ansible_distribution_version|int == 11 %}yes{% else %}no{% endif %}" # SLES11 considers datadog's repo as untrusted
-    state: "{% if datadog_agent5 %}present{% else %}absent{% endif %}"
+# refresh zypper repos only if the template changed
+- name: refresh zypper_repos
```
Comment (olivielpeau) [inline tasks/pkg-suse.yml]: nit on naming consistency with other tasks: name this `Refresh Datadog zypper repo` instead :)

File: tasks/pkg-suse.yml [hunk 2]
```diff
@@ -24,31 +24,30 @@
     state: present
   when: not ansible_check_mode
 
+- set_fact:
+   zypper_repo_url: "{% if datadog_agent5 %}{{datadog_agent5_zypper_repo}}{% else %}{{datadog_zypper_repo}}{% endif %}"
```
Comment (olivielpeau) [inline tasks/pkg-suse.yml]: any reason not to put this logic directly in the template? Or alternatively, set the variable in the `vars` of the next task.

File: tasks/pkg-suse.yml [hunk 3]
```diff
@@ -24,31 +24,30 @@
...
-    name: datadog_5
-    description: Datadog, Inc.
-    repo: "{{ datadog_agent5_zypper_repo }}"
-    enabled: yes
-    auto_import_keys: yes
-    disable_gpg_check: "{% if ansible_distribution_version|int == 11 %}yes{% else %}no{% endif %}" # SLES11 considers datadog's repo as untrusted
-    state: "{% if datadog_agent5 %}present{% else %}absent{% endif %}"
+# refresh zypper repos only if the template changed
+- name: refresh zypper_repos
+  command: zypper refresh
```
Comment (olivielpeau) [inline tasks/pkg-suse.yml]: can we make the command refresh only the `datadog` repository?

File: tasks/pkg-suse.yml [hunk 4]
```diff
@@ -24,31 +24,30 @@
...
-    enabled: yes
-    auto_import_keys: yes
-    disable_gpg_check: "{% if ansible_distribution_version|int == 11 %}yes{% else %}no{% endif %}" # SLES11 considers datadog's repo as untrusted
-    state: "{% if datadog_agent5 %}absent{% else %}present{% endif %}"
+  template:
+    src: zypper.repo.j2
+    dest: /etc/zypp/repos.d/datadog.repo
+    owner: "root"
+    group: "root"
+  register: result
```
Comment (olivielpeau) [inline tasks/pkg-suse.yml]: nit: more specific name than `result` (for example: `datadog_zypper_repo`)

**Review by olivielpeau — APPROVED** [review]
Left a few nits, but overall LGTM as long as it works well 👍 

Feel free to merge once comments are addressed!

File: tasks/pkg-suse.yml (hunk 3 above)
Comment (hush-hush) [reply tasks/pkg-suse.yml]: yes !

**Pull request merged**